tests/matrix2/input.dat

```diff
@@ -0,0 +1,3 @@
+#! An example of using BLAS and LAPACK calls directly from the Psi input file, demonstrating
+#
+assert psi4.core.test_matrix_dpd_interface()
```

Comment:
Have you checked this syntax with a forced fail? I don't think ctest works properly with `assert`, which is a pytest pattern. Try `compare(True, psi4.core.test_matrix..., "matrix dpd")` next.

Reply:
Yes, `ctest` works completely fine as-is. I prefer the current version.

This file works because the Psi input file is parsed to Python. Running the input file raises a Python error which is registered as a crash. This is all standard behavior.